aries-site/src/pages/_document.js

```diff
@@ -36,16 +36,11 @@ export default class MyDocument extends Document {
       <Html lang="en">
         <Head />
         <body>
-          <div id="hpe_slim_header" className="hpe_slim_header" />
+          <div id="header" className="header" />
           <Main />
-          <div id="hpe_slim_footer" className="hpe_slim_footer" />
-          <script src="https://h50007.www5.hpe.com/hfws-static/js/framework/jquery/v-3-5-1/jquery.js" />
-          <script src="https://h50007.www5.hpe.com/hfws/us/en/hpe/slim/root?contentType=js&color_scheme=dark" />
-          <style>
-            {`#hpeslh_site-header .hpeslh_container.hpeslh_wide {
-              border: none;
-            }`}
-          </style>
+          <div id="footer" className="footer" />
+          <script src="https://h50007.www5.hpe.com/hfws-static/js/framework/jquery/v-3-6-0/jquery.js" />
+          <script src="https://h50007.www5.hpe.com/hfws/us/en/hpe/latest.r/root?contentType=js&switchToCHF=true&hide_country_selector=true&hide_search=true&hide_cart=true&hide_sign_in=true&header_logo=hpe&slim_footer=true" />
```

Comment:
Do you see any issues with placing these scripts at the end of the body, rather than in the head as the instructions specify?

I've always tried to push 3rd party scripts as far down as I can, like you are doing here...

Reply:
Same. I did this with the previous one so that the elements are there when the script gets loaded. Not sure it matters here.